packages/tabs/src/Default/Default.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { ThemeProvider } from 'react-bootstrap';
+import Tab from 'react-bootstrap/Tab';
+import Tabs from 'react-bootstrap/Tabs';
+import { TabsDefaultProps } from './Default.types';
+import * as styleMod from './styles.module.scss';
+import { createLocalProps } from '@owlui/utils';
+
+export const Component = (props: TabsDefaultProps) => {
+  const baseClass = 'owluiTabs';
+  const { children, defaultActiveKey, items } = props;
+  const prefix = props.prefix || '';
+
+  const locals = createLocalProps(
+    props,
+    {
+      module: styleMod,
+      classes: {
+        base: baseClass,
+        prefix: prefix,
+        optionals: [
+          {
+            fields: ['theme'],
+            value: 'Theme',
+          },
+          {
+            fields: ['theme', 'appearance'],
+            value: 'Theme',
+          },
+          {
+            fields: ['size'],
+            value: 'Size',
+          },
+        ],
+      },
+    },
+    ['prefix', 'theme', 'appearance', 'size', 'tabs']
+  );
+
+  const [activeKey, setActiveKey] = React.useState<string>(defaultActiveKey);
+
+  return (
+    <ThemeProvider
+      prefixes={{
+        nav: 'owlui-nav',
+        'tab-content': 'owlui-tab-content',
+        'nav-tabs': 'owlui-nav-tabs',
+        'nav-item': 'owlui-nav-item',
+        'nav-link': 'owlui-nav-link',
+        tab: 'owlui-tab',
+        'tab-pane': 'owlui-tab-pane',
+      }}
+    >
+      <Tabs
```

Comment:
You can update the tabs component here to:
```tsx
      <Tabs
        activeKey={activeKey as string}
        onSelect={(k: string | null) => setActiveKey(k)}
        {...locals}
      >
```

Just letting you know, this `as string` is more of a trick to get rid of the TS error. The message I get from TS seems to be related to what I believe is a kind of bug in Bootstrap types 🤔. 
```
Type '(k: string) => void' is not assignable to type 'SelectCallback'.
```